Makefile

```diff
@@ -111,4 +111,9 @@ test-python: generate pyenv${PYTHON_VERSION}
 admin.kubeconfig:
 	hack/get-admin-kubeconfig.sh /subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/${RESOURCEGROUP}/providers/Microsoft.RedHatOpenShift/openShiftClusters/${CLUSTER} >admin.kubeconfig
 
-.PHONY: admin.kubeconfig aro az clean client generate image-aro image-fluentbit image-proxy image-routefix proxy publish-image-aro publish-image-fluentbit publish-image-proxy publish-image-routefix secrets secrets-update e2e.test test-e2e test-go test-python
+vendor:
```

Comment:
I wonder if we want to use `go build -mod=vendor` in this file to enforce that deps that are not checked in to vendor are caught?
General discussion needed around if/how we want to continue to use vendor/
- Protection against go module serving problems? cf https://github.com/gomodules/jsonpatch/issues/28
- This PR seems to mess up my "jump to definition" tooling in vs code, not sure why

Reply:
For now: we'll try to get rid of vendor.go, keep vendor/, add -mod=vendor